back/engines/commercial/multi_tenancy/lib/tasks/core/clean_up_orphaned_image_records.rake

```diff
@@ -25,20 +34,78 @@ namespace :cl2back do
         # is eventually referenced by a layout (when / if the layout is saved).
         # By only destroying unused layout_images with an age of 3+ days, we can be reasonably confident that the
         # admin does not intend to add the image to a layout, and the image is truly orphaned.
-        if image.created_at < 3.days.ago
-          image.destroy! if live_run
-          puts " destroyed layout_image with unused code field: #{image.id}"
-        end
+        next if image.created_at > 3.days.ago
+
+        image.destroy! if live_run
+
+        n_li += 1
+        log = layout_image_log(image, tenant)
+        puts log
+        report["li_#{format('%06d', tot_li + n_li)}"] = log
```

Comment:
Is it possible to move these 4 lines into a method outside of the body of the task? So, the task will be even easier to read.